UKHO.ADM/internal/ConvertTo-GroupObjects.ps1

```diff
@@ -28,16 +38,27 @@ function ConvertTo-GroupObjects {
                 $ggGroup = [ADGroup]::new($Settings.GroupPrefix, $Settings.Environment, $OU.Name, $GroupConfig.Name, "GG", $OU.DistinguishedName, $OU.Domain)
                 $ugGroup.ADGroupMembers += $ggGroup
 
+                # Add Other Groups
+                if ($null -ne $GroupConfig.Groups -and $GroupConfig.Groups.Count -gt 0) {
+                    $GroupConfig.Groups | ForEach-Object {
+                        $groupGroup = [ADGroup]::new($_.DistinguishedName, $_.Domain)
+                        if (Check-GroupExists $groupGroup) {
+                            $ugGroup.ADGroupMembers += $groupGroup
```

Comment:
Should this add to the GG instead of the UG? I am not 100% either way. Looking at our AD on a non primary domain, we add the users to the GG, this is then added to the UG. Does it make more sense for the group to be added to GG so access is "defined in the same place"? Might be worth double checking with @thealmguy

Reply:
probably should go in the GG, as if you added the GG to somewhere, the group would be missed

Reply:
added to GG level